### DISCUSSION TITLE
Redis password not allowed signs

### DISCUSSION THREAD

**Discussion by cavemanr** [mastodon/mastodon#27954] [General]
I have found that if I use a redis password that contains an @, then the login fails and mastodon won't start.

Example password

```
"redis://:_t*jeKZHdQH@9vanoKAnjryQXdkdoV9CJ*cQG_3zVv2.byAzK3UuUAuKs8n@Kvi8zDDp28YFV_M2FwVbdcpkrs*wgjCBQ_.2_FXG@localhost:port/0"
```

**Comment by ThisIsMissEm** [comment]
I believe this is entirely expected, since this cannot be parsed as a connection URL. The redis documentation recommends generating passwords with [`ACL GENPASS`](https://redis.io/docs/latest/commands/acl-genpass/)
